src/ui/dictionary/meaningComponent.svelte

```diff
@@ -12,23 +13,27 @@
     {#each definitions as definition, i}
       <div class="definition">
         {#if definition.definition}
-          <span>Definition:</span>
+          <div class="label">Definition:</div>
           <p>{definition.definition}</p>
         {/if}
         {#if definition.example}
-          <span>Example:</span>
-          <p>{definition.example}</p>
+          <blockquote>
+            {@html 
+              definition.example.replace(
+                new RegExp(`(${word})`, "gi"), 
+                '<i class="mark">$1</i>'
+              )
+            }
+          </blockquote>
         {/if}
         {#if definition.synonyms && definition.synonyms[i]}
           <div class="synonyms">
-            <span>Synonyms:</span>
-            <ul>
-              {#each definition.synonyms as synonym}
-                <li>
-                  {synonym}
-                </li>
+            <div class="label">Synonyms:</div>
+            <p>
+              {#each definition.synonyms as synonym, i}
+                {synonym}{#if i < definition.synonyms.length - 1}{", "}{/if}
```

Comment:
I hope you don't mind this. I've condensed the synonyms down to a comma-separated list to save some space.

Reply:
That's a great improvement!